gdal/frmts/vrt/pixelfunctions.licence

```diff
@@ -0,0 +1,17 @@
+ Permission is hereby granted, free of charge, to any person obtaining a
```

Comment:
I don't see the point of this file . This is the default license of GDAL that is found in the main LICENSE and other file headers